docs/src/mosaic_endpoints.md

```diff
@@ -3,17 +3,17 @@ The `titiler.pgstac` package comes with a full FastAPI application with Mosaic a
 | Method | URL                                                                              | Output                                  | Description
 | ------ | ---------------------------------------------------------------------------------|-----------------------------------------|--------------
 | `POST` | `/mosaic/register`                                                               | JSON ([Register][register_model])       | Register **Search** query
-| `GET`  | `/mosaic/{searchid}/info`                                                        | JSON ([Info][info_model])               | Return **Search** query infos
+| `GET`  | `/mosaic/{search_id}/info`                                                        | JSON ([Info][info_model])               | Return **Search** query infos
```

Comment:
This is to have something similar as `/collections/{collection_id}/items/{item_id}`